### PR TITLE
Finalised all the manual data entry... That really sucked...

### PR DESCRIPTION
**Pull request by AnUnsocialPigeon**
The courses page is now live...

It does use Cataas for the images, as I have yet to be bothered to find a valid image for every course...

Also the data entry really sucked for this... I really should have thought of a better way through doing this but oh well we here now I guess........ xP

### CODE REVIEW

File: src/app/courses/page.tsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    simpleDate: string;
+}
+
+export default async function Home() {
+
+    // Calendar of the course events
+    const currentTime = Date.now();
+
+    // Creating this list has negatively harmed my mental health. Why oh why do I have to do this.
+    // This has made me drastically reconsider how fun a data entry job would be. It went from "not fun" to "i'd rather lick cactuses for a living"
```
Comment (JakubCzarlinski) [inline src/app/courses/page.tsx:19]: ouch oof - summit events were a pain too

File: src/app/meet_the_exec/page.tsx [hunk 2]
```diff
@@ -49,17 +49,13 @@ export default function Home() {
...
-                const otherExec = exec.slice(1);
-
-                // Fisher-Yates shuffle
-                for (let i = otherExec.length - 1; i > 0; i--) {
-                    const j = Math.floor(Math.random() * (i + 1));
-                    [otherExec[i], otherExec[j]] = [otherExec[j], otherExec[i]];
+                // Shhhhh ;) Catification easter egg
+                for (let i = 0; i < exec.length; i++) {
+                    if (Math.random() <= 0.01)
+                        exec[i].PictureURL = "https://cataas.com/cat";
```
Comment (JakubCzarlinski) [inline src/app/meet_the_exec/page.tsx:55]: hell yeah

File: src/components/course_info.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (JakubCzarlinski) [inline src/components/course_info.tsx:1]: haven't used react in a long while - is the required?

Comment (JakubCzarlinski) [reply src/components/course_info.tsx:1]: seems to be unused?

**Review by JakubCzarlinski — APPROVED** [review]

File: src/components/course_info.tsx (hunk 3 above)
Comment (AnUnsocialPigeon) [reply src/components/course_info.tsx:1]: probably not... On it :)

File: src/app/courses/page.tsx (hunk 1 above)
Comment (AnUnsocialPigeon) [reply src/app/courses/page.tsx:19]: ;~;

File: src/components/course_info.tsx (hunk 3 above)
Comment (AnUnsocialPigeon) [reply src/components/course_info.tsx:1]: This has been removed!

**Review by JakubCzarlinski — APPROVED** [review]
LGTM

**Pull request merged**